src/components/parts/frontpage-contact/FrontpageContactAlert.tsx

```diff
@@ -15,7 +15,7 @@ export const FrontpageContactAlert = ({ alertText, yellow = false }: Props) => {
         <div
             className={`${style.alert} ${yellow ? style.yellow : style.white}`}
         >
-            <Warning title="Advarsel" className={style.alertIcon} />
+            <ExclamationmarkTriangleIcon title="Advarsel" className={style.alertIcon} />
```

Comment:
Er det riktig at dette ikke skjules for skjermleser?

Reply:
Nei. Takk.